pkg/apis/tensorflow/v1alpha1/register.go

```diff
@@ -17,6 +17,13 @@ const GroupName = "tensorflow.org"
 // SchemeGroupVersion is the group version used to register these objects.
 var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: CRDVersion}
 
+func init() {
+	// We only register manually written functions here. The registration of the
+	// generated functions takes place in the generated files. The separation
+	// makes the code compile even when the generated files are missing.
+	SchemeBuilder.Register(addDefaultingFuncs)
+}
```

Comment:
Should we add `addKnownTypes` here:

```
SchemeBuilder.Register(addKnownTypes)
```

Reply:
@wackxu ?

Reply:
Do not think so, I think now is better and there is nothing that are confuse.

Reply:
Alright, then LGTM